src/app/const.ts

```diff
@@ -1,5 +1,2 @@
 export const AVALIABLE_LANGUAJES = ['en', 'es'];
-
-// Storage keys
-export const MAIL_KEY = 'Mail_key';
-export const PASS_KEY = 'Pass_key';
+export const SAVED_USER = "Saved user";
```

Comment:
comillas dobles o simples?
```suggestion
export const SAVED_USER = 'Saved user';
```